peers/open_channels.js

```diff
@@ -454,8 +462,8 @@ module.exports = (args, cbk) => {
 
       // Ask for the fee rate to use for internally funded opens
       askForFeeRate: ['isExternal', ({isExternal}, cbk) => {
-        // Exit early when there are no internal funds being spent
-        if (!!isExternal) {
+        // Exit early when there are no internal funds being spent or internal fee rate is specified
+        if (!!isExternal || !!args.internal_fund_fee_rate) {
```

Comment:
isExternal should already be checking for that internal fund so it doesn't need to be check again

Reply:
I guess I'll have to check here or return true on the ` isExternal:` function else it still asks for a funding rate, or you mean something else?

Reply:
If I return true in `isExternal`, then it'll ask for PSBT down below.

Reply:
Ah yeah ok that should be fine then